Add Wikimedia projects

edwardwall opened the pull request:
<!-- Thanks for contributing! Before you submit your pull request, please make sure to check the following boxes by putting an x in the [ ] (don't: [x ], [ x], do: [x]) -->

### Checklist
- [x] The PR isn't documenting something that would be a common practice among password managers (e.g. minimal length of 6)
- [x] The top-level JSON objects are sorted alphabetically
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update.

#### for websites-with-shared-credential-backends.json
- [x] There's evidence the domains are related (SSL certificates, DNS entries, valid links between sites, legal documents etc.)
- [x] The new group serves login pages on each of the included domains, and those login page accept accounts from the others. (For example, we don't associate `google.co.il` to `google.com`, because `google.co.il` redirects to `accounts.google.com` for authentication.)
  
  
These are all Wikimedia projects which share credentials https://www.wikimedia.org/

rmondello commented:
Hi @edwardwall! Can you do a little storytelling here? Are all accounts on all of these websites automatically valid on all of the others? Only some accounts? Is this still the case, or is there some amount of history at play here?

edwardwall commented:
There are some wikimedia.org subdomains which use [non-standard accounts](https://foundation.wikimedia.org/wiki/Privacy_policy/FAQ#standardaccount) and therefore don't share credentials.  
I've removed wikimedia.org and added specific subdomains. Every account created on any of those websites can be used on all the others.  


However there are quite a few other wikimedia.org subdomains which share credentials, eg bd.wikimedia.org and nyc.wikimedia.org. This is [the most complete list I could find](https://meta.wikimedia.org/wiki/Wikimedia_wikis).
